### PR TITLE
Updating autoscaling dashboard to v3

### PR DESCRIPTION
**Pull request by limed**

### CODE REVIEW

File: nubis/puppet/files/grafana/dashboards/aws-auto-scaling_rev3.json [hunk 1]
```diff
@@ -56,7 +56,7 @@
         {
           "aliasColors": {},
           "bars": false,
-          "datasource": "cloudwatch",
```
Comment (gozer) [inline nubis/puppet/files/grafana/dashboards/aws-auto-scaling_rev3.json]: This is a problem and means the dashboard was grabbed from grafana pre-import...

${DS_CLOUDWATCH} won't work, it needs to be 'cloudwatch' verbatim, like it was before

File: nubis/puppet/files/grafana/dashboards/aws-auto-scaling_rev3.json [hunk 2]
```diff
@@ -454,14 +454,14 @@
...
         "current": {},
-        "datasource": "cloudwatch",
+        "datasource": "${DS_CLOUDWATCH}",
         "hide": 0,
         "includeAll": false,
         "label": "AutoScalingGroupName",
         "multi": false,
         "name": "autoscalinggroupname",
         "options": [],
-        "query": "dimension_values($region, AWS/CloudFront, GroupTotalInstances, AutoScalingGroupName)",
```
Comment (gozer) [inline nubis/puppet/files/grafana/dashboards/aws-auto-scaling_rev3.json]: Looks like this is the only fix/change, really.

**Review by gozer — CHANGES_REQUESTED** [review]
Can't have ${DS_*} variables, they need to be literally expanded to 'cloudwatch'

File: nubis/puppet/files/grafana/dashboards/aws-auto-scaling_rev3.json (hunk 1 above)
Comment (limed) [reply nubis/puppet/files/grafana/dashboards/aws-auto-scaling_rev3.json]: Hmm I could have sworn I imported it to grafana and clicked "share" so it should have replaced ${DS_CLOUDWATCH} with "cloudwatch" so not sure why it didn't do it. I'll fix this

**Review by gozer — APPROVED** [review]

**Pull request merged**